From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Carlos A Petry <capetry.dev@gmail.com>
Date: Wed, 7 Nov 2018 23:13:12 -0200
Subject: [PATCH] fpga: altera-cvp: Fix function definition argument

Fix the following checkpatch warning:
WARNING: function definition argument 'struct altera_cvp_conf *'

Signed-off-by: Carlos A Petry <capetry.dev@gmail.com>
Signed-off-by: Moritz Fischer <mdf@kernel.org>
(cherry picked from commit 998c1de56dac7ff6bad4f810259cc25c7d6d2843)
---
 drivers/fpga/altera-cvp.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/fpga/altera-cvp.c b/drivers/fpga/altera-cvp.c
index 770915fb97f9..53b963071c7b 100644
--- a/drivers/fpga/altera-cvp.c
+++ b/drivers/fpga/altera-cvp.c
@@ -57,7 +57,8 @@ struct altera_cvp_conf {
 	struct fpga_manager	*mgr;
 	struct pci_dev		*pci_dev;
 	void __iomem		*map;
-	void			(*write_data)(struct altera_cvp_conf *, u32);
+	void			(*write_data)(struct altera_cvp_conf *conf,
+					      u32 data);
 	char			mgr_name[64];
 	u8			numclks;
 };
-- 
https://clearlinux.org

